webpack/envs/development.ts

```diff
@@ -0,0 +1,78 @@
+import chalk from "chalk"
+import fs from "fs"
+import path from "path"
+import webpack from "webpack"
+import ForkTsCheckerNotifierWebpackPlugin from "fork-ts-checker-notifier-webpack-plugin"
+import ForkTsCheckerWebpackPlugin from "fork-ts-checker-webpack-plugin"
+import FriendlyErrorsWebpackPlugin from "friendly-errors-webpack-plugin"
+import WebpackNotifierPlugin from "webpack-notifier"
+import SimpleProgressWebpackPlugin from "simple-progress-webpack-plugin"
+import { PORT, WEBPACK_DEVTOOL, basePath, isCI } from "../utils/environment"
+
+const cacheDirectory = path.resolve(basePath, ".cache")
+
+if (!isCI && !fs.existsSync(cacheDirectory)) {
+  console.log(
+    chalk.yellow(
+      "\n[!] No existing `.cache` directory detected, initial " +
+        "launch will take a while.\n"
+    )
+  )
+}
+
+export const developmentConfig = {
+  mode: "development",
+  devtool: WEBPACK_DEVTOOL,
+  module: {
+    rules: [
+      {
+        test: /\.styl$/,
+        use: [
+          {
+            loader: "style-loader",
```

Comment:
Haven't fully implemented yet, but in the process of removing our reliance on [stylus middleware](https://github.com/artsy/force/blob/master/src/lib/setup.js#L186-L197)